packages/react-ui/storybook/stories/Introduction.stories.mdx

```diff
@@ -11,12 +11,27 @@ import Storybook from '../assets/storybook.svg';
 # CARTO for React - Widgets and UI Components
 
 This storybook includes the UI Components (@carto/react-ui) for CARTO for React.
-They are based on [Mui](https://mui.com/), and they include a CARTO theme + custom widgets.
+
+## Tech stack
+
+The components are based on [Mui v5](https://mui.com/), and they include a CARTO theme + custom components and widgets.
+
+We have support for `React` 17 & 18.
```

Comment:
Not yet. Even if there are workarounds, we should stick to 17, until we remove all makeStyles. So...

```suggestion
We have currently support for `React` 17 and we're working on having it for v18 asap
```